react-app/src/components/Follow/Follow.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+import { removeFollowing, createFollowing } from '../../store/follow';
+
+import styles from './Follow.module.css';
+
+const Follow = () => {
+    const { userId } = useParams();
+    const dispatch = useDispatch();
+
+    const followings = useSelector(state => state.follow.followings);
```

Comment:
do you need to conditionally check if followings exists like "state.follow?.followings" or will it still work every time

Reply:
It will always be an empty object in the redux state, so I believe it will never error out